brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/RelationshipModifier.java

```diff
@@ -37,7 +41,10 @@ public void apply(EntitySpec<?> entitySpec, NodeTemplate nodeTemplate, Topology
             RelationshipTemplate relationshipTemplate = findRelationshipRequirement(nodeTemplate, requirementId);
             if (relationshipTemplate != null && relationshipTemplate.getType().equals("brooklyn.relationships.Configure")) {
 
-                Map<String, Object> relationProperties = getTemplatePropertyObjects(relationshipTemplate);
+                Map<String, PaaSNodeTemplate> builtPaaSNodeTemplates = treeBuilder.buildPaaSTopology(topology).getAllNodes();
+                String computeName = (nodeTemplate.getName() != null) ? nodeTemplate.getName() : (String) entitySpec.getFlags().get("tosca.template.id");
```

Comment:
`tosca.template.id` should be a constant in `ApplicationSpecsBuilder`. (There are two mentions in this PR and three existing.)